src/Uno.UI/Microsoft/UI/Xaml/Controls/Common/ColorConversion.cs

```diff
@@ -0,0 +1,346 @@
+			// Then, as we transition from yellow and green, green is now completely present, red is becoming decreasingly present, and blue is still not present.
+			// As we transition from green to cyan, green is still completely present, blue is becoming increasingly present, and red is no longer present.  And so on.
+			// 
+			// To convert from hue to RGB value, we first need to figure out which of the three channels is in which configuration
+			// in the sextant that we're located in.  Next, we figure out what value the completely-present color should have.
+			// We know that chroma = (max - min), and we know that this color is the max color, so to find its value we simply add
+			// min to chroma to retrieve max.  Finally, we consider how far we've transitioned from the pure form of that color
+			// to the next color (e.g., how far we are from pure red towards yellow), and give a value to the partially present channel
+			// equal to the minimum plus the chroma (i.e., the max minus the min), multiplied by the percentage towards the new color.
+			// This gets us a value between the maximum and the minimum representing the partially present channel.
+			// Finally, the not-present color must be equal to the minimum value, since it is the one least participating in the overall color.
+			int sextant = (int)(hue / 60); // Uno Doc: Math.Floor() is required after conversion to C# as C++ casts to int differently
+			double intermediateColorPercentage = hue / 60 - sextant;
+			double max = chroma + min;
+
+			double r = 0;
+			double g = 0;
+			double b = 0;
+
+			switch (sextant)
+			{
+				case 0:
+					r = max;
+					g = min + chroma * intermediateColorPercentage;
+					b = min;
+					break;
+				case 1:
+					r = min + chroma * (1 - intermediateColorPercentage);
+					g = max;
+					b = min;
+					break;
+				case 2:
+					r = min;
+					g = max;
+					b = min + chroma * intermediateColorPercentage;
+					break;
+				case 3:
+					r = min;
+					g = min + chroma * (1 - intermediateColorPercentage);
+					b = max;
+					break;
+				case 4:
+					r = min + chroma * intermediateColorPercentage;
+					g = min;
+					b = max;
+					break;
+				case 5:
+					r = max;
+					g = min;
+					b = min + chroma * (1 - intermediateColorPercentage);
+					break;
+			}
+
+			return new Rgb(r, g, b);
+		}
+
+		public static Rgb HexToRgb(string input)
+		{
+			var (rgb, a) = HexToRgba(input);
+			return rgb;
+		}
+
+		public static string RgbToHex(Rgb rgb)
+		{
+			byte rByte = (byte)Math.Round(rgb.R * 255.0);
+			byte gByte = (byte)Math.Round(rgb.G * 255.0);
+			byte bByte = (byte)Math.Round(rgb.B * 255.0);
+
+			UInt64 hexValue = ((UInt64)rByte << 16) + ((UInt64)gByte << 8) + (UInt64)bByte;
+
+			// Uno Doc: Using C#/.net string methods instead
+			string hexString = string.Format("#{0:X6}", hexValue);
+
+			// We'll size this string to accommodate "#XXXXXX" - i.e., a full RGB number with a # sign.
+			//wchar_t hexString[8];
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)